packages/graphiql-react/src/markdown.ts

```diff
@@ -1,3 +1,6 @@
 import MarkdownIt from 'markdown-it';
```

Comment:
I hate to be annoying about tiny things but this would be a tiny but great addition to `@graphiql/toolkit`. perhaps also consider exporting our default config so that folks can create their own instance with custom additional config without adding to the singleton (I've had users talk about wanting to use custom markdown-it plugins for internal docs links/etc!)

An alternative to the singleton could even be customization props for the `MarkdownContent` component and even `GraphiQL` component for the `markdown-it` instantiation.

Reply:
Opened https://github.com/graphql/graphiql/issues/2584 to track this